src/app/@ansyn/plugins/openlayers/open-layers-map/openlayers-map/openlayers-map.ts

```diff
@@ -98,13 +98,13 @@ export class OpenLayersMap extends IMap<OLMap> {
 		this.initMap(element, _mapLayers, position);
 	}
 
-	public positionToPoint(coordinate: ol.Coordinate, cb: (p: GeoJSON.Point) => void) {
+	public positionToPoint(coordinates: ol.Coordinate, cb: (p: GeoJSON.Point) => void) {
```

Comment:
It's a single coordinate, I think the naming here is ok.

Reply:
GeoJSON.Point = {type, coordinates} never `coordinate`